semantic-conventions/README.md

```diff
@@ -92,12 +92,11 @@ semantic conventions that have the tag `network`.
 
 The image supports [Jinja](https://jinja.palletsprojects.com/en/2.11.x/) templates to generate code from the models.
 
-For example, opentelemetry-java [generates typed constants for semantic conventions](https://github.com/open-telemetry/opentelemetry-java/blob/main/semconv/src/main/java/io/opentelemetry/semconv/trace/attributes/SemanticAttributes.java)
-using [this template](https://github.com/open-telemetry/opentelemetry-java/blob/main/buildscripts/semantic-convention/templates/SemanticAttributes.java.j2).
+For example, opentelemetry-java generates typed constants for semantic conventions. Refer to https://github.com/open-telemetry/semantic-conventions-java for all semantic conventions.
 
 The commands used to generate that are
-[here in the opentelemetry-java repo](https://github.com/open-telemetry/opentelemetry-java/blob/main/buildscripts/semantic-convention/generate.sh).
-Note especially the `docker run` commands. For example to generate the aforementioned `SemanticAttributes.java`,
+[here in the opentelemetry-java repo](https://github.com/open-telemetry/semantic-conventions-java)
+Note especially the `docker run` commands. For example to generate a constants file called `SemanticAttributes.java`,
 the following command is used:
```

Comment:
This is no longer the exact command run by Gradle. See [here](https://github.com/open-telemetry/semantic-conventions-java/blob/2be178a7fd62d1073fa9b4f0f0520772a6496e0b/build.gradle.kts#L100-L115). Recommend either removing this part completely (including the code block below) or updating the code block to use the code in the Gradle file (linked earlier).